test/test_model.py

```diff
@@ -2673,7 +2780,9 @@ def test_valid_metrics(self):
             self.assertEqual(fake_script_calls(self, clear=True), expected_calls)
 
     def test_invalid_metric_names(self):
-        invalid_inputs = [
+        invalid_inputs: typing.List[typing.Tuple[
+            typing.Dict[str, float],
+            typing.Dict[typing.Any, typing.Any]]] = [
```

Comment:
I think this is a case where case-based tests *do* improve things: the cases are short and there are a lot of them.

But maybe we can cheat with types here and just use `typing.Any` -- I think the type signature is just getting in our way here.

Reply:
Agreed with the two for secrets; too much noise. I do somewhat like the ones for the list of metric, label pairs. I've cleaned it up a bit by defining an alias and just using that consistently. Work for you?